library/couchdb

```diff
@@ -1,10 +1,14 @@
-# maintainer: Clemens Stolle <klaemo@apache.org> (@klaemo)
+# maintainer: Joan Touzet <wohali@apache.org> (@wohali)
 
-latest: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1
-1.6.1: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1
-1.6: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1
-1: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1
+latest: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 2.1.0
+2.1.0: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 2.1.0
+2.1: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 2.1.0
+2: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 2.1.0
 
-1.6.1-couchperuser: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1-couchperuser
-1.6-couchperuser: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1-couchperuser
-1-couchperuser: git://github.com/klaemo/docker-couchdb@29ed69965ed616a9d0df9a6ffa081773d86c78bc 1.6.1-couchperuser
+1.6.1: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 1.6.1
+1.6: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 1.6.1
+1: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 1.6.1
+
+1.6.1-couchperuser: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 1.6.1-couchperusser
+1.6-couchperuser: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 1.6.1-couchperusser
+1-couchperuser: git://github.com/apache/couchdb-docker@7da42e2725d8ff9b69b78951281f308d8f3fc8f9 1.6.1-couchperusser
```

Comment:
Ah, the build fails since these have an extra "s" on the folder name.  Otherwise this file looks fine.

Reply:
*facepalm* Thanks! Fixed.